components/cli/cmd/cellery/init.go

```diff
@@ -25,12 +25,18 @@ import (
 )
 
 func newInitCommand() *cobra.Command {
+	var projectName = ""
 	cmd := &cobra.Command{
 		Use:   "init",
 		Short: "Initialize a cell project",
-		Args:  cobra.NoArgs,
+		Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
This callback should be only used for validating arguments and returning a proper error. This should check whether the number of arguments is 0 or 1 and return an error otherwise.

You can move the part where you actually get the arguments to the Run function.

Please check other commands and fix in other places as well

Reply:
Updated.